internal/checker/terraform/rules/aws/aws_glue_security_configuration_encrypted.rego

```diff
@@ -0,0 +1,72 @@
+# METADATA
+# title: "Make certain that Glue Security Configuration Encryption is activated"
+# description: "Three AWS glue components, including Cloudwatch, job bookmarks, and S3 buckets, have the potential to be encrypted. This check guarantees that each is properly set."
+# scope: package
+# related_resources:
+# - https://registry.terraform.io/providers/hashicorp/aws/4.8.0/docs/resources/glue_security_configuration
+# custom:
+#   id: CB_TFAWS_107
+#   severity: HIGH
+package lib.terraform.CB_TFAWS_107
+
+import future.keywords.in 
+
+isvalid(block){
+	block.Type == "resource"
+    block.Labels[_] == "aws_glue_security_configuration"
+}
+
+has_attribute (key, value) {
+    _ = key[value]
+}
+
+resource [resource]{
+    block := pass[_]
+	resource := concat(".", block.Labels)
+} 
+resource [resource]{
+    block := fail[_]
+	resource := concat(".", block.Labels)
+} 
+
+pass[block]{
+    block := input[_]
+	isvalid(block)
+    encryptionConfig := block.Blocks[_]
+    encryptionConfig.Type == "encryption_configuration"
+    cloudWatch := encryptionConfig.Blocks[_]
```

Comment:
The `encryptionConfig.Blocks[_]` were declared multiple times kindly use one since it has the same effect